tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Tags.cpp

```diff
@@ -72,15 +72,6 @@ void test_simple_tags() {
   TestHelpers::db::test_simple_tag<
       GeneralizedHarmonic::Tags::ConstraintEnergy<Dim, Frame>>(
       "ConstraintEnergy");
-  TestHelpers::db::test_simple_tag<
-      GeneralizedHarmonic::Tags::GaugeHRollOnStartTime>(
-      "GaugeHRollOnStartTime");
-  TestHelpers::db::test_simple_tag<
-      GeneralizedHarmonic::Tags::GaugeHRollOnTimeWindow>(
-      "GaugeHRollOnTimeWindow");
-  TestHelpers::db::test_simple_tag<
-      GeneralizedHarmonic::Tags::GaugeHSpatialWeightDecayWidth<Frame>>(
-      "GaugeHSpatialWeightDecayWidth");
```

Comment:
Are these tags no longer tested because they're only used in initialization? I would have expected test_simple_tag<> calls some place else after removing them from here, but maybe we just don't need them?

Reply:
I can't test them because they are private and so I don't have access to them. They are effectively tested by testing the initialization action. Does that make sense?